forward_engineering/helpers/azureCLIScriptHelpers/escapeShellSpecialCharacters.js

```diff
@@ -0,0 +1,32 @@
+const getEscapeFunction = shell => {
```

Comment:
# Remark 1
Change this function to have the following signature:

const escapeShellCommandBeforeWrappingInQuotes = (shell, command) => {
     switch (shell) {
          case "powershell":
                return command.replace() ....
     }
}